rpms/SPECS/wazuh-manager.spec

```diff
@@ -22,13 +22,14 @@ Conflicts:   ossec-hids ossec-hids-agent wazuh-agent wazuh-local
 Obsoletes: wazuh-api < 4.0.0
 AutoReqProv: no
 
-Requires: coreutils xz
+Requires: coreutils
 BuildRequires: coreutils glibc-devel automake autoconf libtool policycoreutils-python curl perl
 
 ExclusiveOS: linux
 
 %define _source_payload w9.xzdio
 %define _binary_payload w9.xzdio
+%define _unpackaged_files_terminate_build 0
```

Comment:
Question: Why are you disabling this option?

Reply:
It was a quick testing because I was not finding the real cause of the problem, it was a WIP at that moment.